packages/sourcecred/src/api/bundledGrainIntegrations.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import type {GrainIntegrationFunction} from "../core/ledger/grainIntegration";
+import * as C from "../util/combo";
+
+import {csvIntegration} from "sc-grainIntegration-csv";
+
+export type GrainIntegration = {|
+  name: string,
+  function: GrainIntegrationFunction,
+|};
+
+export function bundledGrainIntegrations(
+  integration: string
+): GrainIntegrationFunction {
+  switch (integration) {
+    case "csv":
+      return csvIntegration;
+    default:
+      throw new Error(
+        "grain integration not found; enter a valid `integration` value in config/grain.json"
+      );
+  }
+}
```

Comment:
This diverges from the pattern used by bundledPlguins and bundledDeclarations. Why?

Reply:
Updated to better approximate the pattern in other bundle files.

I really wanted to have an error message thrown if the config was invalid, since an invalid config would lead to undefined behavior. I thought a switch/case control flow best suited that but I don't mind conforming to the norm. It's also much easier to read all available plugins without reading through all `case`s, I suppose.